llvm/lib/Target/DirectX/DirectXTargetTransformInfo.cpp

```diff
@@ -28,6 +28,8 @@ bool DirectXTTIImpl::isTargetIntrinsicTriviallyScalarizable(
   switch (ID) {
   case Intrinsic::dx_frac:
   case Intrinsic::dx_rsqrt:
+  case Intrinsic::dx_firstbituhigh:
+  case Intrinsic::dx_firstbitshigh:

```

Comment:
you will want to rebase, line ending change made it so a bunch of our prs are getting merge conflcts on this file.